src/tests/test_targets.py

```diff
@@ -43,6 +43,10 @@ def download_iso_override_nodes_count(self, prepared_controller_configuration: B
     def test_target_download_iso(self, cluster):
         cluster.download_image()
 
+    @JunitTestSuite()
+    def test_target_download_files(self, cluster):
```

Comment:
```suggestion
    def test_target_download_ipxe_script(self, cluster):
```
I think the intention would be clearer